models/publication.go

```diff
@@ -61,6 +64,14 @@ func NewPublication(data map[string]rdf.Term) *Publication {
 		descriptionStr := description.String()
 		pub.Description = &descriptionStr
 	}
+
+	if date := data["created"]; date != nil {
+		yearStr := date.String()[0:4]
+		yearInt, err := strconv.Atoi(yearStr)
+		if err == nil {
+			pub.CreatedYear = yearInt
```

Comment:
Do we want to log anything if there are errors here, or would that be more noise than signal?

Reply:
To keep things simple, I'd be inclined to put logging in ETL if there is something that is log-worthy.

Reply:
OK, works for me! Can you whip up a ticket for that?

Reply:
https://github.com/sul-dlss-labs/rialto-etl/issues/251